### PR TITLE
[WIP] Load test for image upload

### PR DESCRIPTION
**Pull request by monikasulik**
Depends on #13, #15 and #16

### CODE REVIEW

File: tests/fixtures/api-db/buyer.json [hunk 1]
```diff
@@ -0,0 +1 @@
...47, "fields": {"created": "2017-01-05T16:20:54.870Z", "modified": "2017-01-05T16:29:18.699Z", "summary": "", "description": "", "employees": "1-10", "export_status": "ONE_TWO_YEARS_AGO", "keywords": "test", "logo": "", "name": "TEST COMPANY", "number": "99999999", "contact_details": {"country": "", "po_box": "", "email_full_name": "Mr Test", "email_address": "test@example.com", "postal_full_name": "", "address_line_2": "", "locality": "London", "address_line_1": "80 Strand", "postal_code": "WC2R 0RL"}, "sectors": ["WATER"], "website": "http://www.example.com", "date_of_creation": "2000-01-24", "is_published": true, "verification_code": "000000000000", "verified_with_code": true, "is_verification_letter_sent": true, "twitter_url": "", "facebook_url": "", "linkedin_url": ""}}, {"model": "user.user", "pk": 2147483647, "fields": {"created": "2017-01-06T17:12:40.092Z", "modified": "2017-01-06T17:12:40.094Z", "sso_id": 2147483647, "name": "", "company": -1, "company_email": "verified@example.com", "company_email_confirmed": false, "company_email_confirmation_code": "345efc53-2c3a-4387-8e5c-70bbd40740fc", "is_active": true, "date_joined": "2017-01-06T17:12:40.092Z", "mobile_number": ""}}]
```
Comment (konradko) [inline tests/fixtures/api-db/buyer.json]: please move to directory-api repo

File: tests/fixtures/api-db/supplier.json [hunk 2]
```diff
@@ -0,0 +1 @@
...ARS_AGO", "keywords": "test", "logo": "", "name": "TEST COMPANY", "number": "99999999", "contact_details": {"country": "", "po_box": "", "email_full_name": "Mr Test", "email_address": "test@example.com", "postal_full_name": "", "address_line_2": "", "locality": "London", "address_line_1": "80 Strand", "postal_code": "WC2R 0RL"}, "sectors": ["WATER"], "website": "http://www.example.com", "date_of_creation": "2000-01-24", "is_published": true, "verification_code": "000000000000", "verified_with_code": true, "is_verification_letter_sent": true, "twitter_url": "", "facebook_url": "", "linkedin_url": ""}}, {"model": "company.companycasestudy", "pk": 2147483647, "fields": {"created": "2016-12-22T14:46:30.658Z", "modified": "2016-12-22T14:46:30.658Z", "title": "Test Case Study", "short_summary": "For load tests", "description": "Test", "sector": "WATER", "website": "http://example.com", "keywords": "test", "image_one": "", "image_two": "", "image_three": "", "image_one_caption": "", "image_two_caption": "", "image_three_caption": "", "video_one": "", "testimonial": "tests", "testimonial_name": "Mr Test", "testimonial_job_title": "QA", "testimonial_company": "DIT", "company": 2147483647}}]
```
Comment (konradko) [inline tests/fixtures/api-db/supplier.json]: ditto

File: tests/__init__.py [hunk 3]
```diff
@@ -22,10 +22,26 @@
     'sso:inactive': 'accounts/inactive/',
     'sso:health': 'api/v1/',
     'sso:user': 'api/v1/session-user/',
+
     # UI-BUYER
     'ui-buyer:landing': '',
     'ui-buyer:register': 'register',
+    'ui-buyer:company-profile': 'company-profile',
```
Comment (konradko) [inline tests/__init__.py]: would probably be nicer to have this module as `utils.py`, putting lots of stuff in ``__init.py`` can be confusing.

Also not sure that this django `reverse` url matching style is better than just hardcoding - would be nice to see what URL is used in the test. Does not need to change now though

File: tests/__init__.py [hunk 4]
```diff
@@ -22,10 +22,26 @@
...
+    'ui-supplier:sectors-health': 'sectors/health',
+    'ui-supplier:sectors-tech': 'sectors/tech',
+    'ui-supplier:sectors-creative': 'sectors/creative',
+    'ui-supplier:sectors-food': 'sectors/food-and-drink',
+    # NOTE: the URLS below require data from fixtures/supplier.json
+    # to be loaded to the API db of the tested system
+    'ui-supplier:suppliers-detail': 'suppliers/00000001',
+    'ui-supplier:case-study': 'case-study/2147483647',
+
     # API
```
Comment (konradko) [inline tests/__init__.py]: `api:sms-verify` is gone, same as `'api:confirm-company-email'`

File: tests/fixtures/sso-db/buyer.json [hunk 5]
```diff
@@ -0,0 +1 @@
+[{"model": "user.user", "pk": 2147483647, "fields": {"password": "pbkdf2_sha256$24000$qPsNN0ZOj885$K+9sqzUeZs80RtsZKsY9L9sT1vDgjaiMlzTcPqrpOa0=", "last_login": null, "is_superuser": false, "created": "2017-01-08T17:56:14.305Z", "modified": "2017-01-08T17:56:14.305Z", "email": "load_tests@example.com", "is_staff": false, "is_active": true, "date_joined": "2017-01-08T17:56:14.275Z", "groups": [], "user_permissions": []}}, {"model": "account.emailaddress", "pk": 2147483647, "fields": {"user": 2147483647, "email": "load_tests@example.com", "verified": true, "primary": true}}]
```
Comment (konradko) [inline tests/fixtures/sso-db/buyer.json]: please move to directory-sso repo, naming / folder structure should be consistent between all

File: tests/__init__.py (hunk 4 above)
Comment (monikasulik) [inline tests/__init__.py]: Good point. I didn't clean up the api ones as AFAIK the api servers are IP-limited and therefore no calls to them work.
That said, are we actually going to be using the api tests? I think last time you said it didn't make that much sense. And also I think it was the most involved in terms of setting up the infrastructure, right? We now have some authenticated tests for sso/buyer. If we need more then it shouldn't be too difficult to add.

**Comment by monikasulik** [comment]
https://pixabay.com/ is where I got the images from. According to the site attribution is not required.

**Pull request merged**